content/guides/building/pre-reqs.md

```diff
@@ -180,8 +180,7 @@ You can now retry the <code>port install...</code> command in a new Terminal.
 
```

Comment:
The alert saying "not a maintained development environment" can be removed as well, I think?